src/files-action.js

```diff
@@ -19,26 +19,51 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+/* eslint-disable */
```

Comment:
that's not a good idea. this will just silence all eslint warnings.

let's address them instead.

`@nextcloud/files` is not a dependency of this app. add it.

Reply:
i was actually trying to dismiss the "import' is only available in ES6", but that was fixed by adding the .jshintrc file. I also added the dependencies that i missed adding 10 times.